rake_nltk/rake.py

```diff
@@ -31,6 +31,8 @@ def __init__(
         punctuations=None,
         language="english",
         ranking_metric=Metric.DEGREE_TO_FREQUENCY_RATIO,
+        max_length=5,
```

Comment:
Why should max length be 5? This number should be a high value to ensure that we do not chop any valid phrase into pieces. According to https://en.wikipedia.org/wiki/Longest_English_sentence, there is no limit on how long a sentence can be, but the longest printed sentence in English is around 14k words (an entire novel)

Even with the assumption, that stop words will definitely come quite frequently in any sensible scenario, we should still keep this limit to a high enough number say 100K just to avoid any possibility of an error. Either way 5 seems like a bad value.

Reply:
At first, I wanted to use the max integer value in python as `Integer.MAX_VALUE` in java. Then I found that there is no fixed max integer value in python, people are using `sys.maxsize` instead, and it differs on different CPUs. Here is a copy of [python sys documentation](https://docs.python.org/3/library/sys.html):

<pre>
sys.<b>maxsize</b>
An integer giving the maximum value a variable of type Py_ssize_t can take. 
It’s usually 2**31 - 1 on a 32-bit platform and 2**63 - 1 on a 64-bit platform.
</pre>

Personally, I feel bad with uncertainty, so I chose a random int 5 in my mind. Yeah, I know this is even much worse than the uncertainty :man_facepalming: , so **I will take your advice 100K as the default value** in my next commit. 

If you have better ideas or find another reliable source, please let me know.